data/GalaxyStellarMassGasMetallicity/conversion/convertStrom2022.py

```diff
@@ -0,0 +1,80 @@
+from velociraptor.observations.objects import ObservationalData
+
+import unyt
+import numpy as np
+import os
+import sys
+
+# Exec the master cosmology file passed as first argument
+with open(sys.argv[1], "r") as handle:
+    exec(handle.read())
+
+output_directory = "../"
+
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+
+# Meta-data
+comment = "The median stellar mass is 1e10 Msun."
+citation = "Strom et al. (2022)"
+bibcode = "2022ApJ...925..116S"
+plot_as = "line"
+redshift = np.array([2.0, 2.7])
+
+M_star = unyt.unyt_array(np.logspace(9.0, 11.3, 100), units="Msun")
```

Comment:
```suggestion
M_star = unyt.unyt_array(np.logspace(9.0, 11.3, 20), units="Msun")
```
With so many points as 100 it is impossible to distinguish individual error bars. 

![stellar_mass_star_fe_over_h_mol_lofloor_50_old](https://user-images.githubusercontent.com/20153933/187068888-248fc696-fc8a-4f91-8d34-ae566d88b982.png)

Le'ts make it 20. Then the error bars will look as follows

![stellar_mass_star_fe_over_h_mol_lofloor_50](https://user-images.githubusercontent.com/20153933/187068877-17b2fc44-d839-4cd5-aa81-0ca79b2de699.png)